app/controllers/navigation_controller.rb

```diff
@@ -0,0 +1,13 @@
+class NavigationController < ApplicationController
+  def recede
+    # Placeholder code for recede action
+  end
+
+  def resume
+    # Placeholder code for resume action
+  end
+
+  def refresh
+    # Placeholder code for refresh action
+  end
+end
```

Comment:
- [ ] As per project [requirements](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/projects/Setup_Controllers_project_v1_1.md#project-requirements), as per the this image:

![image](https://github.com/BANCUNGUYE66/Blog-app/assets/68796232/e645207d-8a7b-46fd-85c5-754381e56d0e)

Please delete what you did here and create 2 controllers with 4 methods, 4 views, and 4 routes one for each method:
- Users controller with index and show methods.
- Posts controller with index and show methods.
don't forget to add a view and route for each method and also don't use `rails generate scaffold` command as you did with the navigation controller.